### ISSUE TITLE
Radio button issue

### ISSUE THREAD

**Issue by dredknight** [Shoes3/shoes3#274]
Hello everyone!

 Its very simple this time... I broke Shoes. Here is a simple code that will show the issue.
When you click on the button and the flow gets refreshed Shoes crashes.
If you comment the "Radio" thing it works fine. Whats going on?

``` Ruby
Shoes.app do
    @a=flow do
        background green
        radio;
    end
    button "refresh" do 
        @a.clear do
            background green
            radio; ### try to comment this one
        end
    end
end
```

**Comment by passenger94** [comment]
Yes, a bug ! Thanks !
`Gtk-CRITICAL **: gtk_radio_button_get_group: assertion 'GTK_IS_RADIO_BUTTON (radio_button)' failed`
works fine with a check button, so something must be wrong in C/Gtk radio code probably ...

**Comment by passenger94** [comment]
Got it !
https://github.com/Shoes3/shoes3/blob/master/shoes/canvas.c#L1568
someone forgot an exclamation point !  ;-)
Should be `if (!NIL_P(shoes_hash_get(canvas->app->groups, group)))` 
exclamation mark before NIL_P is missing.

**Comment by ccoupe** [comment]
Thanks. I've made the change to master.

**Comment by dredknight** [comment]
Awesome :)!

**Comment by ccoupe** [comment]
Closed in Shoes 3.3.2
